core/server/master/src/main/java/alluxio/master/meta/MetaDailyBackup.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.meta;
+
+import alluxio.Configuration;
+import alluxio.Constants;
+import alluxio.PropertyKey;
+import alluxio.master.BackupManager;
+import alluxio.underfs.UfsStatus;
+import alluxio.underfs.UnderFileSystem;
+import alluxio.util.CommonUtils;
+import alluxio.util.FormatUtils;
+import alluxio.util.io.PathUtils;
+import alluxio.wire.BackupOptions;
+import alluxio.wire.BackupResponse;
+
+import com.google.common.base.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.TreeMap;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+
+/**
+ * Backing up primary master metadata everyday at a fixed UTC time.
+ */
+public final class MetaDailyBackup {
```

Comment:
Rename to `DailyMetadataBackup`. The "meta" in `MetaMaster` means "about itself", not "metadata". This is because MetaMaster provides functionality for managing the master itself, as opposed to the filesystem or block storage.

Reply:
I didn't understand the meaning of `MetaMaster` before. Thanks for the explanation!
Changed to `DailyMetadataBackup`.